<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookBackW</class>
 <widget class="QMainWindow" name="BookBackW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>70</y>
      <width>331</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>个人信息</string>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>253</width>
       <height>198</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>读者卡ID：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_reader_id">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="button_search_reader_info">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>姓名：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_reader_name">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>角色：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_reader_role">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>电话：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_reader_phone_num">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_18">
        <item>
         <widget class="QLabel" name="label_33">
          <property name="text">
           <string>账号创建时间：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_create_time">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item>
         <widget class="QLabel" name="label_35">
          <property name="text">
           <string>允许册数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_allow_num">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_20">
        <item>
         <widget class="QLabel" name="label_37">
          <property name="text">
           <string>借出册数：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_has_lent_num">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_9">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>370</y>
      <width>571</width>
      <height>220</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_39">
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>借书记录:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableView_2"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>610</y>
      <width>210</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>70</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>借出</string>
    </property>
    <widget class="QWidget" name="layoutWidget_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>201</width>
       <height>17</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_21">
      <item>
       <widget class="QLabel" name="label_40">
        <property name="text">
         <string>书名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_book_name">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>201</width>
       <height>17</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_22">
      <item>
       <widget class="QLabel" name="label_42">
        <property name="text">
         <string>索取号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_seek_num">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>201</width>
       <height>17</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_23">
      <item>
       <widget class="QLabel" name="label_44">
        <property name="text">
         <string>出版社：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_press_name">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>201</width>
       <height>17</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_24">
      <item>
       <widget class="QLabel" name="label_46">
        <property name="text">
         <string>现存量：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_current_storage">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_14">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>201</width>
       <height>17</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_25">
      <item>
       <widget class="QLabel" name="label_48">
        <property name="text">
         <string>总库存：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_total_storage">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>320</y>
      <width>351</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>图书ID：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_book_id">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_search_book_info">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
